symphony/app/fbcnms-projects/hub/app/components/__generated__/HubVersionQuery.graphql.js

```diff
@@ -4,7 +4,7 @@
  *
  **/
 
- /**
```

Comment:
same with this file - shouldn't be changed

Reply:
Fixed